cmd/clusterctl/client/repository/repository_github_test.go

```diff
@@ -526,3 +523,9 @@ func testMethod(t *testing.T, r *http.Request, want string) {
 		t.Errorf("Request method: %v, want %v", got, want)
 	}
 }
+
+func resetCaches() {
```

Comment:
I wonder if a comment here stating that this is needed to avoid cross-test pollution would be helpful.